### DISCUSSION TITLE
ESP NOW mac address

### DISCUSSION THREAD

**Discussion by tigga2020** [community/community#21437] [Other Feature Feedback, Questions, & Ideas]
<p>I am trying to use ESP NOW to get two NodeMCU ESP32-s devices to communicate.<br>
With the sample code below, it complies using the “0xFF” mac address, however when I change the mac it won’t compile and gives the error quoted below.</p>
<p>My assumption is that uint8_t is not define in the library, but the library does contain an esp_now.h file.<br>
I’m using the Arduino IDE.</p>
<pre><code class="lang-auto">/*
  Rui Santos
  Complete project details at https://RandomNerdTutorials.com/esp-now-esp32-arduino-ide/
      
  Permission is hereby granted, free of charge, to any person obtaining a copy
  of this software and associated documentation files.
  
  The above copyright notice and this permission notice shall be included in all
  copies or substantial portions of the Software.
*/

#include &lt;esp_now.h&gt;
#include &lt;WiFi.h&gt;

// REPLACE WITH YOUR RECEIVER MAC Address
uint8_t broadcastAddress[] = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
</code></pre>
<blockquote>
<pre><code>sketch_oct04a:5:31: error: unable to find numeric literal operator
'operator""C' uint8_t broadcastAddress[] = {8C, AA, B5, A2, 8E, 70};
Multiple libraries were found for "WiFi.h" Used:.. C:\Users######\AppData\Local\Arduino15\packages\esp32\hardware\esp32\1.0.4\libraries\WiFi
Not used: C:\Program Files (x86)\Arduino\libraries\WiFi exit status 1
exponent has no digits</code></pre>
</blockquote>

**Comment by electrojacob** [comment, marked as answer]
<p>Hey tigga2020! Use uint8_t broadcastAdress<span class="chcklst-box fa fa-square-o fa-fw"></span> = {0x8C, 0xAA, 0xB5, 0xA2, 0x8E, 0x70};</p>
